package.json

```diff
@@ -9,10 +9,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@reduxjs/toolkit": "^1.9.1",
+    "@reduxjs/toolkit": "^1.9.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-redux": "^8.0.5"
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.8.0",
+    "react-toastify": "4.1"
   },
   "devDependencies": {
     "@types/react": "^18.0.26",
```

Comment:
We already have yarn.lock so this file is not needed.
It is a bad practice to keep two lockfiles in the same project.

Please use yarn to install dependencies.

Reply:
Okay fine. I will take care these things onwards.